DB/common.php

```diff
@@ -1161,6 +1161,16 @@ function modifyQuery($query)
         $e = new \Civi\Core\Event\QueryEvent($query);
         Civi::$statics['db_common_dispatcher']->dispatch('civi.db.query', $e);
         return $e->query;
+
```

Comment:
@eileenmcnaughton i believe that this was an earlier version of what is in the lines above

Reply:
see https://github.com/civicrm/civicrm-packages/commit/4483923c7c066f71beae92bd3e5e7d25877538ec#diff-e87e96fe24f44eda22f348fa7adc7246